Fix broken references to yew_services (#1861)
yewstack/yew: docs/more/debugging.md

## Panics

The [`console_error_panic`](https://github.com/rustwasm/console_error_panic_hook) crate catches
`panic!`s and outputs them to the console. Yew automatically catches `panic!`s and logs them to

> the browser console.

Reviewer: nit

```suggestion
the browser's console.
```

## Console Logging

> In general, Wasm web apps are able to interact with Browser APIs. The `console.log` API is no 

Reviewer: While we're improving this sentence :D

```suggestion
Wasm web apps can call Browser APIs. The `console.log` API is no 
```

log::info!("Update: {:?}", msg);
```

### [`ConsoleService`](https://docs.rs/yew/latest/yew/services/console/struct.ConsoleService.html)

> This service is included with Yew.

Reviewer: Possible improvement?

```suggestion
This service is defined inside the `yew` crate.
```

## Source Maps

> There is currently no first-class support for source maps for Rust/Wasm web apps. This could change in the future.

Reviewer: Maybe?

```suggestion
Sadly Rust/Wasm web apps are not properly supported by browsers. Hopefully this will change in the future.
```
